.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "MAVProxy/modules/mavproxy_cesium"]
 	path = MAVProxy/modules/mavproxy_cesium
-	url = https://github.com/SamuelDudley/MAVCesium.git
+	url = https://github.com/Ryanf55/MAVCesium.git
```

Comment:
We should clone this into the ArduPilot github set of repos instead, I think.

Reply:
Ok. I don't have admin. Is that something you could set up?